tests/config.rs

```diff
@@ -605,7 +605,7 @@ fn parses_sink_full_es_aws() {
         [sinks.out]
         type = "elasticsearch"
         inputs = ["in"]
-        region = "us-east-1"
+        host = "https://es.us-east-1.amazonaws.com"
```

Comment:
If I understand this correctly, does this mean users can no longer use the regular region names that they did before? If not I see no tests that use that?

Reply:
Yes, there is no need for this since it is represented in the `host` value. This is an `elasticsearch` sink, not an `aws_elasticsearch` sink. All 3 options were very confusing.

Reply:
That is fair, though I did really like how easy it was to send to aws even though it wasn't an aws sink.

Reply:
I don't think it's any less easy. You still need to set the `host` (previously `aws.endpoint`) since each ES cluster gets a unique domain.